randovania/generator/generator.py

```diff
@@ -1,7 +1,7 @@
 import asyncio
 import dataclasses
 from random import Random
-from typing import Callable
+from typing import Callable, Tuple
```

Comment:
we can just use `tuple` in randovania, since we use 3.11 (feature is from 3.10 I believe?)